ocrd_segment/ocrd-tool.json

```diff
@@ -46,6 +74,12 @@
       ],
       "steps": ["layout/analysis"],
       "parameters": {
+        "format": {
+          "type": "string",
+          "enum": ["BMP", "EPS", "GIF", "JPEG", "JP2", "MSP", "PCX", "PNG", "PPM", "TIFF"],
```

Comment:
We could use media types here and have a translation table in ocrd_utils.constants:

```python
MIMETYPE_TO_PIL = {
   "image/png": "PNG", 
  # ...
}
```

Reply:
Yes, one could change the kwargs of `Workspace.add_image_file()` from `format='PNG'` to `mimetype='image/png'`, replace `lower()` [here](https://github.com/OCR-D/core/blob/44556e160a1b93bc1717798c8173cd7980184d2f/ocrd/ocrd/workspace.py#L801) and use `MIMETYPE_TO_PIL` when doing `PIL.Image.save()` and `MIMETYPE_TO_EXT` when setting the `local_filename`.

But then this PR would depend on respective changes in core. I'd say, let's do that in a second step. For now, the above formats do already work.

Reply:
https://github.com/OCR-D/core/issues/439 It's a fairly trivial change in core, I could implement and release that later today if you want.

Reply:
As you please!

Reply:
Is implemented in core v2.4.0